src/dmd/parse.d

```diff
@@ -8483,6 +8483,11 @@ final class Parser(AST) : Lexer
         auto e = parseCondExp();
         while (1)
         {
+            // require parens for e.g. a ? a : a = a;
```

Comment:
This example doesn't seem to be in the test cases.

Reply:
I've now updated this line to be equivalent to a test case.